config/routes.rb

```diff
@@ -0,0 +1,10 @@
+Rails.application.routes.draw do
+  devise_for :users, path: '/'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  root 'splash#index'
+
+  resources :groups do
+    resources :exchanges
+  end
```

Comment:
- [Optional] Hi @tchasinga, you are using a resourceful routes here without filtering out the implemented actions. The resourceful routing declare by default all the common routes for a given resource(groups and exchanges) while you didn't implement all the actions. This reduces the efficiency of your app. It is important to always add necessary routes :+1: